### PR TITLE
Changed the text in slides in the hero to be terser. Issue #408

### PR DESCRIPTION
**Pull request by dengjonathan**
Additionally changed the structure of the text to be 2 distinct header elements, rather than a break within once header element.

I believe this is more correct HTML style and also creates a visual distinction between the headline and the subheadline.

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -5,27 +5,22 @@
       <li class="valign-wrapper">
         <%= image_tag 'gi-bill.jpg' %>
         <div class="caption center-align valign">
-          <h3>
-            Welcome back. You served your country.<br />
-            Now let's get you coding and to your next mission.
-          </h3>
+          <h3>Thank you for serving</h4>/>
+          <h4>Now let's get you coding and on to your next mission</h4>
```
Comment (hollomancer) [inline app/views/pages/home.html.erb]: Unsure if we want to remove the punctuation at the end. Opinions?

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -167,9 +160,9 @@
         <h5 class="center">Employer Services</h5>
 
         <p class="light">
-          Beyond providing feedback on your resumé, we provide intros to software
-          companies to get your foot in the door and pursue a career in a fast-paced
-          and exciting industry that is growing by the day.
+          It's not just a resumé class. We provide intros to software
+          companies that will allow you to pursue a career in a fast-paced
+          and growing industry.
```
Comment (hollomancer) [inline app/views/pages/home.html.erb]: I really like the simplified copy here. :+1:

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -184,8 +177,7 @@
 
         <p class="light">
           Our in-house web development team employs military veterans to build
-          websites and web applications while returning profits back into our
-          nonprofit organization to build organizational capacity.
+          websites and web applications for real companies.
```
Comment (hollomancer) [inline app/views/pages/home.html.erb]: This is much better. :+1:

**Comment by hollomancer** [comment]
@davidmolina, thoughts?

**Comment by davidmolina** [comment]
I <3 that's it more clear, concise. Ship it.

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (rickr) [inline app/views/pages/home.html.erb]: I don't care either way but it should be consistent. @dengjonathan can you go through and make sure each item either has punctuation or doesn't?

**Comment by rickr** [comment]
Have you seen how the `h4`s look on a live site?

**Comment by dengjonathan** [comment]
I'll need to take a look.

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -5,27 +5,22 @@
       <li class="valign-wrapper">
         <%= image_tag 'gi-bill.jpg' %>
         <div class="caption center-align valign">
-          <h3>
-            Welcome back. You served your country.<br />
-            Now let's get you coding and to your next mission.
-          </h3>
+          <h3>Thank you for serving</h4>/>
```
Comment (hollomancer) [inline app/views/pages/home.html.erb]: I think you may have meant `<h3>Thank you for serving</h3>` here.

**Comment by rickr** [comment]
Hi @dengjonathan, do you have a moment to address the comments in this PR?

**Comment by rickr** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**